types/rest/client.d.ts

```diff
@@ -7,7 +7,7 @@ export function validateSignature(uri: string, nonce: string, signature: string,
  * You can also pass in additional parameters accepted by the node requests module.
  */
 export class Client {
-    constructor(authId?: string, authToken?: string, options?: string);
+    constructor(authId?: string, authToken?: string, options?: any);
```

Comment:
@narayana-plivo I believe the same line change we need to update in the **client-test.d.ts** file right?

Reply:
@kalyan-plivo currently its passed with  constructor(authid: string, authToken: string, proxy: string); Not object.